.github/workflows/ci.yml

```diff
@@ -280,7 +280,8 @@ jobs:
           else
             # e.g. push event to branch called myaccount-patch-1
             # use git history to make a best attempt at getting the parent branch
-            PARENT_BRANCH=$(git show-branch -a 2>/dev/null | grep '\*' | grep -v `git rev-parse --abbrev-ref HEAD` | head -n1 | sed 's/.*\[\(.*\)\].*/\1/' | sed 's/[\^~].*//')
+            # note: { grep * || :; } is to prevent exit code of 1 on zero-match from halting workflow
+        PARENT_BRANCH=$(git show-branch -a 2>/dev/null | { grep '\*' || :; } | { grep -v `git rev-parse --abbrev-ref HEAD` || :; } | head -n1 | sed 's/.*\[\(.*\)\].*/\1/' | sed 's/[\^~].*//')
             export COMPOSER_ROOT_VERSION="${PARENT_BRANCH}.x-dev"
```

Comment:
Is it okay for `COMPOSER_ROOT_VERSION` to be `.x-dev` in the event that we can't figure out the parent branch? Seems like we should have some fallback, or an appropriate error message and explicit exit 1.

Reply:
That's a good point, have updated to fallback to the current branch